### PR TITLE
Minor grammar fixes in README.md

### PR DESCRIPTION
**Pull request by Chenkail**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)
- [x] None of the above

#### Description of the changes in your PR

- Minor grammar fixes in README.md

#### Fixes the following issue(s)

- None

#### APK testing 

No changes made to the app

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ When a change to the NewPipe code occurs (due to either adding features or bug f
  3. Download the APK from [Github Releases](https://github.com/TeamNewPipe/NewPipe/releases) and install it as soon as we publish a release.
  4. Update via F-droid. This is the slowest method of getting updates, as F-Droid must recognize changes, build the APK itself, sign it, then push the update to users.
 
-We recommend method 2 for most users. APKs installed using method 2 or 3 are compatible with each other, but not with those installed using method 4. This is due to the same signing key (ours) being using for 2 and 3, but a different signing key (F-Droid's) being used for 4. Building a debug APK using method 1 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app.
+We recommend method 2 for most users. APKs installed using method 2 or 3 are compatible with each other, but not with those installed using method 4. This is due to the same signing key (ours) being used for 2 and 3, but a different signing key (F-Droid's) being used for 4. Building a debug APK using method 1 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app.
```
Comment (FireMasterK) [inline README.md:90]: This change is grammatically incorrect. `used` is the correct word to use here.

**Review by FireMasterK — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mhmdanas) [reply README.md:90]: I don't see anything wrong. They changed `using` to `used`, which, like you said, seems correct to me.

Comment (FireMasterK) [reply README.md:90]: My bad, I read the change the wrong way around 👍

**Review by TobiGr — APPROVED** [review]
Thanks

**Pull request merged**